### PR TITLE
prepare release

### PR DESCRIPTION
**Pull request by rogalmic**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
 0.1.1
 =====
 ## [Bugfix release v0.1.1](https://github.com/rogalmic/vscode-bash-debug/releases/tag/v0.1.1)
+- new launch parameters: program, args, cwd, pathBash, pathBashdb, pathCat, pathMkfifo, pathPkill, trace
```
Comment (wataash) [inline CHANGELOG.md]: Could you enclose variables in backticks for visibility (example: compare vscode-python's [raw markdown](https://raw.githubusercontent.com/Microsoft/vscode-python/release-2018.2/CHANGELOG.md) and one on vscode) ? like:
``
`pathBash`
``

File: src/bashDebug.ts [hunk 2]
```diff
@@ -285,7 +289,8 @@ export class BashDebugSession extends LoggingDebugSession {
 				const frameLine = parseInt(lineContent.substr(lineContent.lastIndexOf(" ")));
 
 				if ((process.platform === "win32")) {
-					frameSourcePath = this.getWindowsPathFromLinux(frameSourcePath);
+
+					frameSourcePath = frameSourcePath.startsWith("/") ? reverseWSLPath(frameSourcePath) : reverseWSLPath(`${this.workingDirectory}/${frameSourcePath}`);
 				}
```
Comment (rogalmic) [inline src/bashDebug.ts]: Needs testing/changes for non-windows OS

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
+- [BREAKING]deprecated parameters: scriptPath -> program, commandLineArguments -> args, bashDbPath -> pathBashdb, bashPath -> pathBash
 - fix default launch.json generation after breaking change in Microsoft's debug adapter interface
-- various fixes
+- fix pkill usage on macOS
 - dependency update to mirror vscode-mock-debug
```
Comment (wataash) [inline CHANGELOG.md]: I don't think this line and "update icon to png" are informative for users.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,9 +1,14 @@
 0.1.1
 =====
 ## [Bugfix release v0.1.1](https://github.com/rogalmic/vscode-bash-debug/releases/tag/v0.1.1)
+- new launch parameters: program, args, cwd, pathBash, pathBashdb, pathCat, pathMkfifo, pathPkill, trace
+- [BREAKING]deprecated parameters: scriptPath -> program, commandLineArguments -> args, bashDbPath -> pathBashdb, bashPath -> pathBash
 - fix default launch.json generation after breaking change in Microsoft's debug adapter interface
-- various fixes
+- fix pkill usage on macOS
```
Comment (wataash) [inline CHANGELOG.md]: Could you remove this line for now?
I want write one more fix for pkill path handling on macOS.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,9 +1,14 @@
...
 - dependency update to mirror vscode-mock-debug
+- dynamic default pathBash generation, allowing 32bit and 64bit VSCode usage on Windows
+- update icon to png (svg deprecated in package.json)
+- automated deployment (possibility to download pre-release extension's vsix directly from github)
```
Comment (wataash) [inline CHANGELOG.md]: Here's my draft for change logs:

```
- update attributes in `launch.json`
  - *BREAKING* change attribute names
    - `bashDbPath` -> `pathBashdb`
    - `bashPath` -> `pathBash`
    - `scriptPath` -> `program`
  - add `cwd` attribute to set working directory (#25, #32)
  - add `trace` attribute to enable logging of the Debug Adapter Protocol
  - fix default `launch.json` generation after breaking change in Microsoft's debug adapter interface (#39, #41, #45, #46)
- dynamic default `pathBash` generation, allowing 32bit and 64bit VSCode usage on Windows
- automated deployment (possibility to download pre-release extension's vsix directly from github)
```

Adopt all or part of this if you want.

File: src/bashDebug.ts (hunk 2 above)
Comment (wataash) [reply src/bashDebug.ts]: Let me test on macOS.

**Comment by rogalmic** [comment]
Changelog updated - thanks for review!

Can you take a look at the conversion to absolute path on other OS? (I commented it in PR)

File: src/bashDebug.ts [hunk 6]
```diff
@@ -92,6 +94,8 @@ export class BashDebugSession extends LoggingDebugSession {
 			args.program = `${getWSLPath(args.program)}`;
 		}
 
+		this.workingDirectory = args.cwd;
```
Comment (wataash) [inline src/bashDebug.ts]: Isn't `this.launchArgs.cwd` enough?

File: src/extension.ts [hunk 7]
```diff
@@ -47,6 +47,12 @@ class BashConfigurationProvider implements vscode.DebugConfigurationProvider {
 			return undefined;
 		}
 
+		if (!folder)
+		{
```
Comment (wataash) [inline src/extension.ts]: Please join them as `if (!folder) {` if you use Shift+Alt+F.

**Comment by wataash** [comment]
Works well on macOS :+1:

**Comment by wataash** [comment]
GIF in #38 was not intended to be included in README.md, sorry for misleadingness.
Can I force push to delete 1aa9cb7?
We should delete 1aa9cb7 since `images/bash-debug.gif` is non-negligible large (1.3 MiB).

**Comment by rogalmic** [comment]
Sure, just porvide correct one...

2018-03-13 16:08 GMT+01:00 Wataru Ashihara <notifications@github.com>:

**Comment by rogalmic** [comment]
Looks good. Nice trick with the blob for image!

**Comment by wataash** [comment]
Thanks :)
I'll create PR to handle macOS `pkill` problem tomorrow...

File: src/bashDebug.ts (hunk 6 above)
Comment (rogalmic) [reply src/bashDebug.ts]: Yes it is - good catch.

**Pull request merged**